build.py

```diff
@@ -574,7 +577,9 @@ def create_dockerfile_buildbase(ddir, dockerfile_name, argmap):
             zlib1g-dev \
             libarchive-dev \
             pkg-config \
-            uuid-dev && \
+            uuid-dev \
+            libnuma-dev \
+            libnuma1 && \
```

Comment:
Do we need libnuma1 if we have libnuma-dev?

Reply:
Yeah.